### ISSUE TITLE
[Feature Request] Just the Logic Tree/Map portion and not the wiki?

### ISSUE THREAD

**Issue by wakatara** [Kungsgeten/org-brain#216]
**TLDR**

Can I have just the single file concept mapping portion of this or disable The Brain functions?... =] 

**Longer Tale**

I was looking for a text based Logic Tree tool and Yes/No Trees more than, say... your use case here of implementing The Brain as a concept mapped tree of all knowledge.

Is there a way to just have each .org file give you as a logic tree map without the holistic overview of the entire mind mapping wiki space? (and also as a left to right tree that is ascii formatted like the 'tree' command... =]  

Btw, I have to admit to being really impressed by your lisp-fu. Great work.  =]

**Comment by Kungsgeten** [comment]
Hi! I don't quite understand what you mean here. What part is it that you want to exclude? If its just that you don't want to see the text of entries when visualizing, that's possible by setting `org-brain-show-text` to `nil`. There's also a package named org-mind-map: https://github.com/theodorewiles/org-mind-map maybe that's similar to what you want (I don't think a node can have more than one parent there though)?

**Comment by wakatara** [comment]
Heya!

Apologies if I was confusing (and thanks for the quick response!)

So, imagine org-brain and org-mind-map had a baby. =p  I really like the fact yours has a text visualize interface (rather than using graphviz) but like the fact that org-mind-map has the graphs limited to one file and goes from left to right. 

I guess what I'm asking is it possible to get that mash-up of those from options in org-brain? 

And then in my mind I had thought it might be clearer if it worked left -> right. Somewhat like tree though obviously rejigging as nodes added in org-mode:

```
.
├── People_aware
│   ├── People_aware_and_been
│   │   ├── People_consistently_attending
│   │   └── People_non-returning
│   └── People_aware_and_not_been
└── People_unaware
```

lemme know if that's not clearer. It certainly seems close to the core of what you have. Just limited to a file and having display more logic tree or yes/no tree oriented. And yes, I am obviously trying to do too much in org-mode... =p

**Comment by wakatara** [comment]
(in fact, now I'm writing this down and looking at it in the interface... wondering if something like org-bullets is probably a better start to riff off this for trying to accomplish this. Hmmmm....   Though I did want to have the capability to visualize and have diff levels of info available. =]

**Comment by Kungsgeten** [comment]
Maybe something like [org-sidebar](https://github.com/alphapapa/org-sidebar) could be helpful?

**Comment by alphapapa** [comment]
@Kungsgeten I'm afraid I don't understand any better than you do.  :)

**Comment by bepolymathe** [comment]
I'am curious about that...

**Comment by Kungsgeten** [comment]
@wakatara I have now tried `org-sidebar` (see previous link) and if I understand what you want correctly then I think it does what you want. I'm closing this issue since I think the "one file tree view" is outside the scope of `org-brain` and better suited towards other packages (like `org-sidebar`).
